mne/viz/evoked.py

```diff
@@ -172,7 +172,8 @@ def _plot_legend(pos, colors, axis, bads, outlines, loc, size=30):
     ax = inset_axes(axis, width=str(size / ratio) + '%',
                     height=str(size) + '%', loc=loc)
     ax.set_adjustable("box")
-    pos_x, pos_y = _prepare_topomap(pos, ax, check_nonzero=False)
+    pos_x, pos_y = pos.T
+    _prepare_topomap(pos, ax, check_nonzero=False)
```

Comment:
this is weird. You assign vars and then use them 2 lines later. Please swap the 2 new lines.